app/client/src/sagas/WidgetOperationSagas.tsx

```diff
@@ -1035,6 +1035,9 @@ function* addTableWidgetFromQuerySaga(action: ReduxAction<string>) {
       parentRowSpace: 1,
       parentColumnSpace: 1,
       isLoading: false,
+      props: {
+        tableData: `{{${queryName}.data}}`,
+      },
```

Comment:
```suggestion
      props: {
        tableData: `{{${queryName}.data}}`,
        dynamicBindings: {
          tableData: true
        }
      },
```
If you do this you will not have to call the `updateWidgetPropertyRequest` done the line